tests/test_api/conftest.py

```diff
@@ -89,6 +92,21 @@ def make_db_record(job_id,
     return record
 
 
+def setup_requests_mock_with_actual_polygons(granule_polygon_pairs):
+    cmr_response = {
+        'feed': {
+            'entry': [
+                {
+                    'producer_granule_id': granule,
+                    'polygons': polygons
+                } for granule, polygons in granule_polygon_pairs
+            ]
+        }
+    }
+    responses.reset()
+    responses.add(responses.POST, CMR_URL_RE, json.dumps(cmr_response))
```

Comment:
Any thoughts on having `setup_requets_mock()` now call `setup_requests_mock_with_actual_polygons()` to eliminate some of the duplication?

Reply:
Done, let me know what you think. Renamed `actual` -> `given` so as not to imply that they must be the actual polygons for the granules.